data/genomics/mus_musculus/rnaseq_expression/SRP254919.gene_meta.tsv

```diff
@@ -0,0 +1,1001 @@
+gene_id	gene_name
+ENSMUSG00000000305	Cdh4
+ENSMUSG00000000308	Ckmt1
```

Comment:
Could we have a shorter list for this?

Reply:
This is designed to match the expression matrix. It's for use in downstream applications that process that matrix alongside the row (gene) and column (sample metadata). I can 'fake it' in each application, but then I have to keep repeating the faking.